device/celestica/x86_64-cel_silverstone-r0/th3-128x100G.config.bcm

```diff
@@ -0,0 +1,441 @@
+pbmp_xport_xe.0=0x8ffff8ffffcffff8ffff8ffff8ffffcffff9fffe

```

Comment:
This seems to be the same as "Silverstone-128x100/th3-128x100G.config.bcm ", if not needed, please remove.

Reply:
if you will get config file from hwsku,  we will remove it.

Reply:
fixed in 2e3ce7e